astropy/units/tests/test_quantity.py

```diff
@@ -1132,7 +1131,7 @@ def test_insert():
     assert q2.unit is u.m
     assert q2.dtype.kind == 'f'
 
-    if NUMPY_VERSION >= version.LooseVersion('1.8.0'):
+    if minversion(np, '1.8.0'):
```

Comment:
This one is numpy >= 1.8, so no.  This was the only place that was checked for so I didn't define a variable for it.